app/websockets/handlers.py

```
@@ -1,28 +1,54 @@
 import json
+import uuid
 
 from fastapi import WebSocket
 
-from .manager import WebSocketManager
-from .schemas import ChatMessage
+from .enums import AGUIEvent, Role
```

Ruff (0.12.2)

`.enums.Role` imported but unused

Remove unused import: `.enums.Role`

(F401)